lms/envs/common.py

```diff
@@ -3127,6 +3127,7 @@
 
 ENTERPRISE_ENROLLMENT_API_URL = LMS_ROOT_URL + "/api/enrollment/v1/"
 ENTERPRISE_PUBLIC_ENROLLMENT_API_URL = ENTERPRISE_ENROLLMENT_API_URL
+ENTERPRISE_COURSE_ENROLLMENT_AUDIT_MODES = ['audit']
```

Comment:
We'd also need it to be present in `lms/envs/aws.py` so that it can load a custom preference from JSON.